extensions/data-operation/data-operation-actions/src/main/java/org/apache/unomi/operation/router/EventKafkaContextProducer.java

```diff
@@ -0,0 +1,105 @@
+package org.apache.unomi.operation.router;
+
+import org.apache.camel.ProducerTemplate;
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.component.jackson.JacksonDataFormat;
+import org.apache.camel.component.kafka.KafkaComponent;
+import org.apache.camel.component.kafka.KafkaConfiguration;
+import org.apache.camel.component.kafka.KafkaEndpoint;
+import org.apache.camel.core.osgi.OsgiDefaultCamelContext;
+import org.apache.unomi.api.Event;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.BundleEvent;
+import org.osgi.framework.SynchronousBundleListener;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+
+public class EventKafkaContextProducer implements SynchronousBundleListener, EventProducer {
+    private OsgiDefaultCamelContext camelContext;
+    private BundleContext bundleContext;
+    protected String kafkaTopic = "eventify-event";
+    private ProducerTemplate producer;
+    private JacksonDataFormat objectMapper;
+    private static Logger logger = LoggerFactory.getLogger(EventKafkaContextProducer.class);
+
+    private Map<String, String> kafkaProps;
+
+    public void initCamelContext() throws Exception {
```

Comment:
I don't think we need Camel here, regarding the route, it could be implemented directly.

It introduces lot of dependencies for not lot of gain.

More other, the camelcontext should be registered as service (to be visible by camel:* commands).